app/keygen/cmd/decrypt.go

```diff
@@ -37,7 +37,7 @@ func rootKeyTriplet(content string) (string, string, string) {
 }
 
 func keys() (string, string, string) {
-	p := env.KeyGenRootKeyPath()
```

Comment:
I renamed the ones for `Safe` because `Safe` was looking like an adjective in say `SafeSpiffeId()` implying spiffie id is safe/secure/etc; which is not the case; the spiffe id is an id destined to safe it is a `SpiffeIdForSafe()`

Once renaming Safe env getters, the rest were inconsistent; so I renamed these too, for consistency and alignment.